lib/internal/fs/utils.js

```diff
@@ -722,12 +733,28 @@ const validateRmOptionsSync = hideStackFrames((path, options) => {
       throw err;
     } else if (err.code === 'ENOENT' && !options.force) {
       throw err;
+    } else if (warn && err.code === 'ENOENT') {
+      emitPermissiveRmdirWarning();
     }
   }
 
   return options;
 });
 
+let permissiveRmdirWarned = false;
+
+function emitPermissiveRmdirWarning() {
+  if (!permissiveRmdirWarned) {
+    process.emitWarning(
+      'Permissive rmdir recursive is deprecated, use rm recursive and force \
+instead',
+      'DeprecationWarning',
+      'DEP0147'
```

Comment:
@Trott @BridgeAR what should we do here, since we won't be assigned the `DEP0147` until we release the 14.x.x doc deprecation?

Reply:
It was supposed to get assigned when it landed on master but...uh...that didn't happen.  Follow-up fast-track PR to assign it similar to 37321a9e11f2198 should resolve all this.